bmlab/controllers.py

```diff
@@ -531,6 +533,76 @@ def extract_payload_spectra(self, image_key):
         evm.set_spectra(image_key, spectra)
         return spectra, times, intensities
 
+    def get_maps(self, parameter_key):
```

Comment:
The plural in the function name indicates that it returns multiple maps, but in fact it only returns one map, correct?

Reply:
Squeezing is fine if the dimensionality is always returned. I have no preference here. Unsqueezing also does not cost anything.